language/docs/usage.adoc

```diff
@@ -326,6 +326,59 @@ If you decided to go with the `--json-output` option, it means output will consi
 * `errors` field is an array of strings
 # TODO log field
 
+== Writing a Rudder technique
+
+Rudder language techniques can be compiled to a Json file which can be imported in Rudder in the technique editor or using the API.
+Note that rudder language and the technique editor are not yet fully compatible. Some features will only be available in pure rudder language (without importing the technique to Rudder)
+and some will only be available by creating a technique through the GUI.
+
+In the restricted case where you want to import a rudder language technique to the technique editor, the known limitations are:
+
+* Only one resource by technique, which is the technique (as understand by the technique editor)
+* A state declaration in rudder language equals to a method call in the technique editor
+* Resulting conditions must be manually set and equals to the default ones from the technique editor
+* Technique resources can not yet be defined this way
+* The resource describing the technique must be prefixed by `technique_`
+
+==== Example
+
+Let's write a very simple technique in rudder language:
+
+.technique.rd
+[source, language]
+----
+@format=0
+@name="Deploy vim" // <1>
```

Comment:
// is not recognised as a comment character by rudderc, so we can't copy paste the example